frame/balances/src/lib.rs

```diff
@@ -468,7 +468,7 @@ decl_module! {
 			let who = T::Lookup::lookup(who)?;
 			let existential_deposit = T::ExistentialDeposit::get();
 
-			let wipeout = new_free + new_reserved < existential_deposit;
+			let wipeout = new_free.saturating_add(new_reserved) < existential_deposit;
```

Comment:
DQ regarding your earlier comment offline: If a chain panics in a normal transactions, would that be super bad (I don't mean for Polkadot per se, in general)? it tells us clearly that something in this module is broken. and as long as there is an update path (most of the time: we won't panic in `on_initialize()`) then it is actually okay-ish, kinda? 🤔

Reply:
It means a user can spam the chain with transactions which would cause panic and never be charged a fee.

Reply:
ahh right 👍 .